[2024-12-04 17:27:19] INFO - train: Starting training process...
[2024-12-04 17:27:19] INFO - train: Loading configurations...
[2024-12-04 17:27:19] INFO - train: Using device: cuda
[2024-12-04 17:27:19] INFO - train: Creating data loaders...
[2024-12-04 17:28:00] INFO - train: Setting up models and trainer...
[2024-12-04 17:28:00] INFO - train: Initializing distillation training...
[2024-12-04 17:28:04] INFO - train: Starting training...
[2024-12-04 17:28:04] ERROR - train: An error occurred: 'dict' object has no attribute 'training'
Traceback (most recent call last):
  File "/nas2/jieui/class/deepnlp/project/scripts/train.py", line 97, in main
    trainer.train()
  File "/nas2/jieui/class/deepnlp/project/src/trainers/base_trainer.py", line 68, in train
    for epoch in range(self.config.training.num_epochs):
AttributeError: 'dict' object has no attribute 'training'
